packages/account/src/Sections/Security/ConnectedApps/connected-apps-info-bullets.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Text } from '@deriv/components';
+import { CONNECTED_APPS_INFO_BULLETS } from 'Constants/connected-apps-config';
+
+type TConnectedAppsInfoBulletsProps = {
+    class_name_dynamic_suffix?: string;
+    text_color?: string;
+    text_size: string;
+};
+
+const ConnectedAppsInfoBullets = ({
+    class_name_dynamic_suffix,
+    text_color,
+    text_size,
+}: TConnectedAppsInfoBulletsProps) => (
+    <ol
+        style={{ fontSize: `var(--text-size-${text_size})` }}
```

Comment:
Why don't you use the style file to handle the text size? we should not have inline css changes!

Reply:
we can actually create `ol` element via `Text` component, using `as` prop. thanks to @likhith-deriv. 🌷 
this inline styling is deleted since prop can be used to mention the size.